src/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+LABEL maintainer="Antonio Tauro <toni.tauro@adfinis.com>"
+
+ADD back8sup.sh /
+
+ENV KUBECTL_VERSION=v1.17.0
+
+RUN chmod +x /back8sup.sh && apk add --no-cache bash jq curl py3-pip && \
+    curl -L -o/usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl && \
+    pip3 install yamllint yq &&\
+    adduser -D backup
```

Please only run one `&&` joined command per line and also add ` --no-cache-dir` to the pip call.

Personally i prefer formatting RUN statements as follows, but that's really just me being opinionated af...
```suggestion
RUN chmod +x /back8sup.sh \
  && apk add --no-cache bash jq curl py3-pip \
  && curl -L -o/usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl \
  && chmod +x /usr/local/bin/kubectl \
  && pip3 --no-cache-dir install yamllint yq \
  && adduser -D backup
```